### DISCUSSION TITLE
elasticserch cluster config

### DISCUSSION THREAD

**Discussion by ghost** [trinodb/trino#10002] [Q&A] [answered]
In elasticsearch.properties, if I set `"elasticsearch.host=host1,host2,host3"` it will report "Name or service not known", change the set to `"elasticsearch.host=host1"` works fine, any advise?

**Comment by ebyhr** [comment, marked as answer]
The `elasticsearch.host` configuration property doesn't accept multiple hosts. Elasticsearch connector tries to discover other nodes using `/_nodes/http` request internally.
